.github/workflows/npm.yml

```diff
@@ -109,6 +109,13 @@ jobs:
             fi
             pushd "${bin}"
             sed -i "s/\"version\": \".*\",/\"version\": \"${node_version}\",/" package.json
+            for os in linux darwin
+            do
+              for arch in x86 arm64
```

Comment:
Something is not aligned. According to the PR description we should put `x64` instead of `x86` here. Also the package.json looks as if we have used `x64`.

Reply:
damn. thank you!